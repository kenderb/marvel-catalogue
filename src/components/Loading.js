import React from 'react';
import { IonSpinner } from '@ionic/react';

const SpinnerStyle = {
  margin: 'auto',
  transform: 'scale(5)',
  color: 'red',
};
const Loading = () => (<IonSpinner name="bubbles" style={SpinnerStyle} />);

export default Loading;
