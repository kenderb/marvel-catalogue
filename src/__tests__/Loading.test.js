import { create } from 'react-test-renderer';
import Loading from '../components/Loading';

test('Loading renders correctly', () => {
  const tree = create(<Loading />).toJSON();
  expect(tree).toMatchSnapshot();
});
