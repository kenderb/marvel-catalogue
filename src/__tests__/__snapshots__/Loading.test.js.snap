// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<h1>
  Loading...
</h1>
`;
